### ISSUE TITLE
The Choose Cloud Platform dialog is not accessible to screen readers

### ISSUE THREAD

**Issue by Oire** [KoenZomers/KeePassOneDriveSync#28]
Hello,  
KeePass is renowned for its accessibility to screen reader users (blind and visually impaired). More than that, there is a special setting that improves its interoperability with screen readers.  
Your plugin seems to have great functionality but I couldn't try it for the Choose Cloud Platform dialog is not accessible. You have here:

```csharp
        public OneDriveCloudTypeForm()
        {
            InitializeComponent();

            OneDriveConsumerPicture.Image = Resources.OneDriveConsumer;
            OneDriveForBusinessPicture.Image = Resources.OneDriveForBusiness;
        }
```

A blind person like me cannot click on such images, they are not presented in the tab order nor in the accessibility object tree.  
What I propose is either to change them to radio buttons or, if this is not possible by design, to make them accessible at least for screen reader compatibility mode.

Thanks.

**Comment by KoenZomers** [comment]
Hi Oire,

Thanks for bringing this to my attention! I'll investigate options for improving that screen. Would be nice if you can test it you see of it works well. Will update this thread once it's available for testing.

**Comment by Oire** [comment]
Thanks a lot! If you decide what to do, I might even investigate and make a patch. C# is not my strongest language, but it would be great for me to try :).

**Comment by KoenZomers** [comment]
Hi Oire,

I've replaced the images with two buttons having a tab order on them. Can you download the plugin specifically from this location and try it out to see if this works better for you:

https://github.com/KoenZomers/KeePassOneDriveSync/tree/ImpairedImprovements

**Comment by Oire** [comment]
Thanks a lot, it works now!

**Comment by KoenZomers** [comment]
Thanks for testing and pointing it out @Oire! I've merged it to the master branch for release.
